crates/re_ui/examples/re_ui_example.rs

```diff
@@ -1,4 +1,33 @@
-use re_ui::{toasts, Command, CommandPalette, CommandReceiver, CommandSender};
+use re_ui::{toasts, CommandPalette, UICommand, UICommandSender};
+
+/// Sender that queues up the execution of a command.
+pub struct CommandSender(crossbeam::channel::Sender<UICommand>);
+
+impl UICommandSender for CommandSender {
+    /// Send a command to be executed.
+    fn send_ui(&self, command: UICommand) {
+        // The only way this can fail is if the receiver has been dropped.
+        self.0.send(command).ok();
+    }
+}
+
+/// Receiver for the [`CommandSender`]
+pub struct CommandReceiver(crossbeam::channel::Receiver<UICommand>);
+
+impl CommandReceiver {
+    /// Receive a command to be executed if any is queued.
+    pub fn recv(&self) -> Option<UICommand> {
+        // The only way this can fail (other than being empty)
+        // is if the sender has been dropped.
+        self.0.try_recv().ok()
+    }
+}
+
+/// Creates a new command channel.
+fn command_channel() -> (CommandSender, CommandReceiver) {
+    let (sender, receiver) = crossbeam::channel::unbounded();
```

Comment:
We can just use `std::sync::mpsc::channel` here - it uses `crossbeam` internally anyways. We really only need `crossbeam` for bounded channels.